packages/orbit-components/src/ErrorFormTooltip/Tooltip/index.tsx

```diff
@@ -76,6 +73,7 @@ const ErrorFormTooltip = ({
       id={id}
       ref={refs.setFloating}
       aria-live="polite"
+      aria-hidden={!shown}
```

Comment:
I was debating whether to add any `role` and change/remove the `aria-live` or alternatively open this prop. From my research, `role="tooltip"` is never used with the `aria-live` so I didn't add the tooltip role. I would say that we should keep the `aria-live` here, at least for the `error` one. I was thinking about adding the `role="alert"`, however, with this role the `aria-live="assertive"` is connected - which could also be an option for the `error` 😶 I am open to the ideas here.

Reply:
From my usage, it seems to be working good as is, so I wouldn't touch it until someone more expert suggests otherwise. It is correctly announced by screen readers when the input element is focused, so I think that's fine by now. WDYT?

Reply:
Okay, sounds good to me 👍